components/StoryCards/StoryCard.module.scss

```diff
@@ -93,22 +101,19 @@ $small-card-image-size-desktop: 164px;
 .small {
     @include tablet-up {
         display: grid;
-        grid-template-columns: $small-card-image-size auto;
+        grid-template-columns: $small-card-image-width auto;
         gap: 0 $spacing-4;
         align-items: center;
     }
 
     @include desktop-up {
-        grid-template-columns: $small-card-image-size-desktop auto;
+        grid-template-columns: $small-card-image-width auto;
```

Comment:
Was this changed from Hugo's input? 🤔 

Honestly, I don't like how the images are now looking stretched:
<img width="1339" alt="image" src="https://user-images.githubusercontent.com/1641218/156346574-46bc653e-a8d5-46db-82e8-b7988a802761.png">

I believe that square images look much better. I also remember discussing it with Hugo and he agreed that square images are better 
(@iamhugofelix please correct me if I'm wrong or if you changed your mind on that 🙏🏻 )
<img width="1191" alt="image" src="https://user-images.githubusercontent.com/1641218/156346739-fb34dd15-e06a-4c1c-ada9-41be13bc0a70.png">

Reply:
I was just following the Figma file.

I agree that square images look better but I am not sure if it's gonna work because now with categories, title, subtitle and date all being possibly visible there's not enough space.

Just look at the Cern story and how big the title is and it's not even rendering a subtitle.

@iamhugofelix Should we keep using the dimensions that we've had so far?

Reply:
We should use the square image every time it is possible. The only reason for stretching it is for space purposes. Check how it should behave ⬇

![Uploading CleanShot 2022-03-02 at 09.16.35@2x.png…]()
️

Reply:
Okay I've changed the image size back to the original value but used `min-height` and `height: 100%` so that the image stretches vertically if needed.

It now looks like this:
![Screenshot 2022-03-02 at 13 39 40](https://user-images.githubusercontent.com/4209081/156363236-6eb2f6fd-fe85-4a4d-84d9-cbc6137afbc5.png)

Reply:
That's perfect, thanks!